### ISSUE TITLE
Add button to depleted drinks

### ISSUE THREAD

**Issue by dieechtenilente** [AhiruSoftwareCompany/alcoholtestapp#76]
Some of us don't want to track all of their drinking habits.. strange.

**Comment by okitec** [comment]
So is this about clearing depleted drinks from the timeline? If so, I'd make it a setting like "keep last X drinks", "keep all drinks", "delete once depleted" instead of adding a per-drink button the user has to press manually. Or is this about hiding certain drinks, in which case the user betrays themselves?

Maybe I've misunderstood something, haven't used the app for obvious reasons

**Comment by dieechtenilente** [comment]
@okitec Yes, I want to add a way to get rid of all depleted drinks. For now, a drink can be deleted by long pressing a drink item, which is in some cases not ideal. But these are some nice ideas, thanks!
